### ISSUE TITLE
Can't resize neovim-gtk window! Missing window borders!

### ISSUE THREAD

**Issue by Zardoz89** [daa84/neovim-gtk#60]
Possible related to https://github.com/daa84/neovim-gtk/issues/58#issuecomment-366181843

Looks that neovim-gtk window not have borders where I can click and drag to resize the window, except on the title bar.

![imagen](https://user-images.githubusercontent.com/888483/36416108-601b8392-1628-11e8-803c-ceed25815909.png)

I try with commit f9b20db , Kubuntu 17.10 + kubuntu backports + dark breeze gtk theme

**Comment by daa84** [comment]
In gnome border exists. Looks like this also depends on wm.

**Comment by Zardoz89** [comment]
Some weeks ago, It had borders (thin, but there was). It was broke on some point. I could try to find on what commit this changed.

**Comment by paldepind** [comment]
I have this issue as well. This seems to be caused by the client side decorations. It would be really nice to have an option to disable CSD. In my experience CSD never works nicely outside of GNOME.

**Comment by Zardoz89** [comment]
I agree with @paldepind . I think that is related to CSD. Before I begin to use kubuntu backports, the Neovim-gtk CSDs was working nice. Some change on Qt or kwin from the backports, should had break partially the CSD borders. 
I suggest to add a option to disable CSD and/or control the border size. 

Sadly I don't have idea of rust or about gtk+ stuff related to CSDs, to try to help more.

**Comment by daa84** [comment]
If this is CSD problem, you can try to disable titlebar by setting environment variable NVIM_GTK_NO_HEADERBAR=1, maybe this helps.

**Comment by Zardoz89** [comment]
I just installed gtk3-nocsd and launch neovim-gtk with it. The borders are back! So yes, the problem is the CSD
